cloudcannon.config.yaml

```diff
@@ -1,11 +1,70 @@
+data_config:
+  blog-tags: true
 collections_config:
   pages:
+    icon: feed
+    disable_add_folder: true
+    output: true 
+    path: content
+    schemas:
+      page: 
+        path: schemas/page.html
     _enabled_editors:
       - visual
-  blog:
+  articles:
+    icon: article 
+    path: content/blog
+    disable_add_folder: true
+    schemas:
+      post: 
+        path: schemas/post.md
+    add_options:
+      - name: Add Article
+        editor: content
+        schema: post
     _enabled_editors:
       - content
       - visual
+  meta_data:
```

Comment:
Metadata is one word, so this should be `metadata` imo

Reply:
Or just `data`

Reply:
i'll rename this to data